server/server.js

```diff
@@ -252,6 +254,13 @@ app.get('/dashboard/:installationId?', async (req, res) => {
   }
 })
 
+app.get('/buttons-data', async (req, res) => {
+  const data = await db.getDataForExport()
+
+  const csv = csvParser.parse(data)
```

Comment:
I'm not super worried about special characters because this uses a library, but it would probably still be worth putting a double-quote in the `notes` column and making sure that it exports it properly (I would expect it to put double-double-quotes (i.e instead of `Hello"World` it would do `Hello""World`).

Reply:
Tried this locally and it still works!